ofproto/ofproto-dpif.c

```diff
@@ -3337,6 +3335,7 @@ bundle_set(struct ofproto *ofproto_, void *aux,
 
         bundle->floodable = true;
         bundle->protected = false;
+        bundle->p4_bridge_id = s->p4_bridge_id;
```

Comment:
I believe this needs to be bracketed by `#if defined(P4OVS)` / `#endif`.

Reply:
Ahh yes, i have fixed it.